pubspec.yaml

```diff
@@ -19,5 +20,109 @@ dev_dependencies:
     sdk: flutter
   flutter_lints: ^1.0.0
   test: ^1.17.0
+#  golden_toolkit: 0.12.0
```

Comment:
💅 Please remove code that is commented out.

Reply:
oh, thanks for the nice catch 👌